### DISCUSSION TITLE
Type converter from kotlinx.datetime.LocalTime to java.time.LocalTime? is not found for column

### DISCUSSION THREAD

**Discussion by hantsy** [Kotlin/dataframe#887] [Q&A] [answered]
Upgraded to `v0.14.0`, got exception of the column type `java.time.LocalTime`.

**Comment by Jolanrensen** [comment]
Hi! Could you give an example of which operation you're trying to perform? I might have missed a conversion doing this PR https://github.com/Kotlin/dataframe/pull/843 but I want to be sure :)

**Comment by hantsy** [comment]
Given an example like this. 

```kotlin
data class (
   @ColumnName("Start")
   val startedTime:LocalTime // java 8 DateTime
)
```
Which works well in the previous version.

Now I have to add a manual convert to make it work. 

```kotlin
   dataFrame
   .convert{ "Start"<kotlinx.datetime.LocalTime>()}.with{
        it.toJavaLocalTime()
   }
   
```
 
Even though we are using Kotlin in our backend, Java 8 Datetime is more popular than Kotlin DateTime. 

We do not want to introduce more dependencies that we do not use.

**Reply by Jolanrensen** [reply, marked as answer]
Ah I see, yes it was an oversight on my end. I changed the default datetime classes to kotlinx everywhere (because it was a bit mixed before) but I forgot to check automatic conversion was possible between the two.
You should still be allowed to use the java ones if you like of course, so I 'll make an issue to add the missing converters from kotlinx LocalTime to Java Localtime (and more). Thanks for letting us know!

**Reply by Jolanrensen** [reply]
https://github.com/Kotlin/dataframe/issues/894
